### PR TITLE
added nav bar and carousel

### PR DESCRIPTION
**Pull request by slackerbarra**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  background-image: url("../img/wood-2.jpg");
+}
+
+.item.carousel-wood-img img {
+  height: 250px;
+  margin: 100px 0;
+}
+
+.carousel-img-two {
+  background-image: url("../img/restaurant-2.jpg");
```
Comment (ryekerjh) [inline css/styles.css:31]: ouch...you didnt follow my conventions :(

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+.navbar-inverse {
+    background-color: transparent;
+    border-color: transparent;
+}
+
+.navbar-inverse .navbar-nav>.active>a, .navbar-inverse .navbar-nav>.active>a:focus, .navbar-inverse .navbar-nav>.active>a:hover {
+  background-color: rgba(8, 8, 8, 0.6);
+}
```
Comment (markrogers9109) [inline css/styles.css:11]: Nice specific targeting!

File: index.html [hunk 3]
```diff
@@ -9,7 +9,80 @@
...
+	        <span class="icon-bar"></span>
+	      </button>
+	      <a class="navbar-brand" href="#"><img src="img/name-01.png" alt="" class="img-responsive"></a>
+	    </div>
 
+	    <!-- Collect the nav links, forms, and other content for toggling -->
+	    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+	      <ul class="nav navbar-nav navbar-right">
+					<li class="active"><a href="#">Home <span class="sr-only">(current)</span></a></li>
+					<li><a href="#">Link</a></li>
```
Comment (ryekerjh) [inline index.html]: Change these from `link` to their proper names

File: index.html [hunk 4]
```diff
@@ -9,7 +9,80 @@
...
+	    <li data-target="#carousel-example-generic" data-slide-to="2"></li>
+	  </ol>
+
+  <!-- Wrapper for slides -->
+	  <div class="carousel-inner" role="listbox">
+	    <div class="item active">
+				<div class="item active carousel-flex carousel-wood-img">
+		      <img src="img/logo-01.png" alt="...">
+		      <div class="carousel-caption">
+		        ...
```
Comment (ryekerjh) [inline index.html]: Remove these (...)

**Review by ryekerjh — COMMENTED** [review]
Take a look at these , fix the things I mentioned and then add, commit and push again

**Review by markrogers9109 — APPROVED** [review]
Good Job!

**Pull request merged**